.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: false
+language: go
+go:
+- 1.7
+- 1.8
+- 1.9
+- tip
+install:
+- go get github.com/golang/dep
```

Comment:
I believe this should be `go get github.com/golang/dep/cmd/dep`?

Reply:
Indeed, but we decided to just use the binary as per https://github.com/golang/dep/blob/master/docs/FAQ.md#how-do-i-use-dep-in-ci